src/EventSubscriber/PageHeaderSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\localgov_guides\EventSubscriber;
+
+use Drupal\node\Entity\Node;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\localgov_core\Event\PageHeaderDisplayEvent;
+
+/**
+ * Class PageHeaderSubscriber.
+ *
+ * @package Drupal\localgov_guides\EventSubscriber
+ */
+class PageHeaderSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      PageHeaderDisplayEvent::EVENT_NAME => ['setPageHeader', 0],
+    ];
+  }
+
+  /**
+   * Set page title and lede.
+   */
+  public function setPageHeader(PageHeaderDisplayEvent $event) {
+    if ($event->getEntity() instanceof Node &&
+      $event->getEntity()->bundle() == 'localgov_guides_page' &&
+      $event->getEntity()->localgov_guides_parent
+    ) {
+      $overview = $event->getEntity()->localgov_guides_parent->entity;
```

Comment:
If it's set, so a entity field list, but an empty one does this return null?
(ie a localgov guide page without the parent set - I assume it's compulsory but)